.github/workflows/test.yml

```diff
@@ -9,15 +9,15 @@ on:
 
 env:
   # Even though we can test against multiple versions, this one is considered a target version.
-  TARGET_GOLANG_VERSION: "1.18"
+  TARGET_GOLANG_VERSION: "1.19"
```

Comment:
I think its good to keep this as the minimum version, as go versions are fully backwards compatible it makes sense to test against the lowest rather than latest.

Reply:
Sgtm